recipe/meta.yaml

```diff
@@ -1,15 +1,7 @@
-{% set name = "or-tools" %}
-{% set package_name = "ortools" %}
 {% set version = "9.2" %}
-{% set home = "https://developers.google.com/optimization" %}
-{% set license = "Apache-2.0" %}
-{% set license_family = "Apache" %}
-{% set license_file = "LICENSE-2.0.txt" %}
-{% set dev_url = "https://github.com/google/or-tools" %}
-
 
 package:
-  name: {{ name|lower }}-package
+  name: or-tools
```

Comment:
I would like to keep this in case we ever need that name.

Reply:
In an multi-output recipe, the name of the "package" is irrelevant and doesn't show up anywhere. But fine by me to re-add.

Reply:
Yes, more about keeping the recipe clean should the name ever be used.